src/pyhf/infer/calculators.py

```diff
@@ -667,7 +667,11 @@ def __init__(
 
     def distributions(self, poi_test, track_progress=None):
         """
-        Probability Distributions of the test statistic value under the signal + background and background-only hypothesis.
+        Probability distributions of the test statistic value under the signal + background and background-only hypotheses.
+
+        These distributions are produced by generating MC toys with the nuisance parameters set to their conditional
+        maximum likelihood estimators at the corresponding value of the parameter of interest for each hypothesis, following
+        the recommendations in https://indico.cern.ch/event/126652/contributions/1343592/attachments/80222/115004/Frequentist_Limit_Recommendation.pdf.
```

Comment:
Does this exist anywhere more stable than the Indico entry that Eilam added? Is it publicly on CDS anywhere?

Reply:
it otherwise exists on ATLAS internal twiki pages... pretty sure it's not on CDS

Reply:
Okay. I'll look and I'll also make some small edits to the docstrings so that the URL gets rendered in the docs nicely.

c.f. https://pyhf--1610.org.readthedocs.build/en/1610/_generated/pyhf.infer.calculators.ToyCalculator.html#pyhf.infer.calculators.ToyCalculator.distributions

Reply:
Ah even better, this is covered in the joint ATLAS and CMS "[Procedure for the LHC Higgs boson search combination in Summer 2011](https://cds.cern.ch/record/1379837/)".

> Generate toy Monte Carlo pseudo-data to construct pdf s f(˜qµ|µ, ˆθ obs µ ) and f(˜qµ|0, ˆθ obs 0 ) assuming a signal with strength µ in the signal+background hypothesis and for the background-only hypothesis (µ = 0). These distributions are shown in Fig. 1. Note, that for the purposes of generating a pseudo-dataset, the nuisance parameters are fixed to the values ˆθ obs µ or ˆθ obs 0 obtained by fitting the observed data, but are allowed to float in fits needed to evaluate the test statistic. This way, in which the nuisance parameters are fixed to their maximum likelihood estimates, has good coverage properties [12].